README.md

```diff
@@ -33,6 +33,111 @@ Get the latest version [here](https://github.com/llorllale/cactoos-matchers/rele
 
 Java version required: 1.8+.
 
+## How to use
+
+Use our matchers inside your JUnit [`@Test`](http://junit.sourceforge.net/javadoc/org/junit/Test.html) case. We also provide `Assertion<T>` in which you can compose the behavior under test and the test case's matcher, and attempt to affirm it.
+
+We provide matchers for several different domains:
+
+### Text
+Examples:
+
+```java
+@Test
+public void textHasPrefix() {
+  final String prefix = "Application startup";
+  new Assertion<>(
+    "must have the prefix",
+    () -> new TextOf(new File("some.log")),
+    new StartsWith(prefix)
+  ).affirm();
+}  
+
+@Test
+public void csvLineHasCorrectFormat() throws Exception {
+  final String fields = "^[^|]+|[^|]+|[^|]+$";
+  new Assertion<>(
+    "must match the expected pattern",
+    () -> new FirstOf<>(
+      line -> true,
+      new SplitText(
+        new TextOf(new File("report.csv")),
+        "\n"
+      )
+    ).value(),
+    new MatchesRegex(fields)
+  ).affirm();
+}
+```
+
+### Concurrency
+Examples:
+
+*RunsInThreads*
```

Comment:
The tests related to this matcher are unstable, therefore I wouldn't recommend to publish it now.
This info might be added once the #51 is done.

Reply:
@dgroup I don't see anything wrong with `RunsInThreads`. The issues we've been having (such as #51) have more to do with the test setup than anything else.

Reply:
Ok.